spec/ruby/core/dir/chroot_spec.rb

```diff
@@ -3,7 +3,7 @@
 require File.expand_path('../shared/chroot', __FILE__)
 
 platform_is_not :windows do
-  not_supported_on :jruby do
+  not_supported_on :jruby, :truffleruby do
```

Comment:
Ok thanks, I'll leave it up to you then. I've added 2 more cases I've missed in optional/capi.